ql/src/java/org/apache/hadoop/hive/ql/parse/ColumnStatsSemanticAnalyzer.java

```diff
@@ -48,10 +41,15 @@
 import org.apache.hadoop.hive.serde2.typeinfo.PrimitiveTypeInfo;
 import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
 import org.apache.hadoop.hive.serde2.typeinfo.TypeInfoUtils;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.hadoop.hive.ql.metadata.HiveUtils.unparseIdentifier;
+
```

Comment:
It's generally better not to shuffle imports to minimize the chances of conflicts between PRs. 

I know it's the IDE and that is painful to double check and put them back each time, but unless we do a pass of refactoring and we put a lint check on the import order, it's better not to change it unless necessary.